src/main/java/Duke.java

```diff
@@ -1,10 +1,143 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("____________________________________________________________");
+        System.out.println("Hello! I'm Sura");
+        System.out.println("What can I do for you?");
+        System.out.println("____________________________________________________________");
+
+        Scanner scanner = new Scanner(System.in);
+        Task[] tasks = new Task[100];
+        int taskCount = 0;
+
+        while (true) {
+            String userInput = scanner.nextLine();
+
+            if ("bye".equalsIgnoreCase(userInput)) {
+                break;
+            }
+            if ("list".equalsIgnoreCase(userInput)) {
+                System.out.println("____________________________________________________________");
+                System.out.println("Here are the tasks in your list:");
+                for (int i = 0; i < taskCount; i++) {
+                    Task task = tasks[i];
+                    System.out.println((i + 1) + "." + task);
+                }
+                System.out.println("____________________________________________________________");
```

Comment:
Could implement SLAP here such that code fragment won't have multiple levels of abstraction.